src/core/activity_queue.rs

```diff
@@ -167,7 +168,8 @@ async fn do_send(
         }
         Ok(o) => {
             let status = o.status();
-            let text = o.text().await.map_err(Error::conv)?;
+            // Limit the status text to 1KB.
```

Comment:
Also mention that this is for security reasons. Could also use the same 100 KB constant from below for simplicity.